client/my-sites/stats/controller.jsx

```diff
@@ -437,3 +438,9 @@ export function wordAds( context, next ) {
 
 	next();
 }
+
+export function emailOpen( context, next ) {
```

Comment:
I see other routes doing 

`	if ( 0 === siteId ) {
		window.location = '/stats';
		return next();
	}
`
Is this something that can be added here as well? Or is this something you plan on adding later?

Reply:
Indeed, that will be there when the siteId is required by the component. Thanks for the review! 🏅